### PR TITLE
Fix job detail run history integration tests

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Adjust the default sort behaviour and fix the job run history integration tests.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

**Comment by Poltergeist** [comment]
image

🚢  LGTM!

File: tests/_fixtures/metronome/job.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
         }
       ]
     }
-  ] ,
```
Comment (pierlo-upitup) [inline tests/_fixtures/metronome/job.json]: i wonder if we should ignore this change – not a biggie but cypress keeps changing these fixtures every time it runs and it's pretty annoying...

**Comment by pierlo-upitup** [comment]
If the fixture change does not bother you, 🚢

File: tests/_fixtures/metronome/job.json (hunk 1 above)
Comment (orlandohohmeier) [inline tests/_fixtures/metronome/job.json]: What do you recommend? Should I drop that change?

Comment (pierlo-upitup) [inline tests/_fixtures/metronome/job.json]: As you please.

Comment (orlandohohmeier) [inline tests/_fixtures/metronome/job.json]: Ok.. I'll not change it, as the cypress versions (0.16.4) I'm using, seem to be is ok with it.

**Pull request merged**